.config/igo/config.toml

```diff
@@ -0,0 +1 @@
+port = "9999"
```

Comment:
the location for this file would be in the users home directory, so I don't think we need to commit this file

Reply:
@sowmiyamuthuraman looks like this file still stuck around can we remove it it?